src/plots/cartesian/dragbox.js

```diff
@@ -601,32 +601,35 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
             subplots = Object.keys(plotinfos);
 
         for(var i = 0; i < subplots.length; i++) {
+
             var subplot = plotinfos[subplots[i]],
+                clipId = 'clip' + fullLayout._uid + subplots[i] + 'plot',
```

Comment:
This feels pretty frail.

Maybe we could store a subplot clip id in `fullLayout._plots[i].clipId`